### PR TITLE
[RM CH04] Add Edge Profile in CH04

### PR DESCRIPTION
**Pull request by ASawwaf**
Fixes https://github.com/cntt-n/CNTT/issues/1721

Add Edge in Cloud Infrastructure Profiles in RM CH04

### CODE REVIEW

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -410,6 +410,9 @@ This Cloud Infrastructure Profile is intended to be used for both IT workloads a
 #### 4.2.4.2 Network Intensive Profile
 This Cloud Infrastructure Profile is intended to be used for those applications that has high network throughput requirements (up to 50Gbps).
 
+#### 4.2.4.3 Edge Profile
+This Cloud Infrastructure Profile is intended to be used for Edge use cases that have special requirmenet , as FPGA for vRAN use case which required offloading some RAN functionality.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter04.md:414]:
```suggestion
This Cloud Infrastructure Profile is intended to be used for Edge use cases that have special requirements, as FPGA for vRAN use case which required for offloading some RAN functionality.
```

Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter04.md:414]: I think FPGA is not a requirement, but a solution to a requirement. The requirement here is low latency computing, low latency networking and high throughput.

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter04.md:414]: Suggestions for some Typo corrections
```suggestion
This Cloud Infrastructure Profile is intended to be used for Edge use cases that have special requirements, such as for the vRAN use case where  FPGA is required for offloading some RAN functionality.
```

Comment (pgoyal01) [reply doc/ref_model/chapters/chapter04.md:414]: @CsatariGergely Please review my suggestion and see if it addresses your concern.

**Comment by rabi-abdel** [comment]
@pgoyal01 @walterkozlowski @CsatariGergely can you please re-review

**Comment by walterkozlowski** [comment]
@rabi-abdel, I am happy to approve once the typos are fixed by @ASawwaf as per the text proposed by @pgoyal01:

This Cloud Infrastructure Profile is intended to be used for Edge use cases that have special requirements, as FPGA for vRAN use case which required for offloading some RAN functionality.

**Comment by ASawwaf** [comment]
thanks @ulikleber for this catch :)

**Comment by rabi-abdel** [comment]
@ASawwaf please advice what would you like to do with this PR. based on the conclusion of last meeting to add extensions to NI profile, shall this PR be closed now?

**Comment by ASawwaf** [comment]
Thanks for your contribution as agreed last week, we will look on Profile capabilities in ch04 and figure out which one will be used in Edge use cases so no need for edge profiles

**Pull request closed without merging**